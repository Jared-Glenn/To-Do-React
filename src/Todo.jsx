import React from "react";

const Todo = ({ id, item, deleteTodo }) => {
    return (
        <li>
            <span>{ item }</span>
            <button onClick={ deleteTodo } data-testid="todo">X</button>
        </li>
    )
};

export default Todo;