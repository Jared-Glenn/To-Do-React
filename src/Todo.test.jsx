import { render } from "@testing-library/react";
import Todo from "./Todo";

// Smoke Test
it('renders without crashing', () => {
    render(<Todo />);
});

// Snapshot Test
it('matches the snapshot', () => {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
});