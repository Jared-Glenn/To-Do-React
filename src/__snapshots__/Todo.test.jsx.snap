// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <li>
    <span />
    <button
      data-testid="todo"
    >
      X
    </button>
  </li>
</DocumentFragment>
`;
